includes/db.inc

```diff
@@ -35,9 +35,14 @@ function islandora_usage_stats_pid_id($pid) {
  */
 function islandora_usage_stats_pid_to_db($pid) {
   $object = islandora_object_load($pid);
-  return db_insert('islandora_usage_stats_objects')
-    ->fields(array('pid' => $pid, 'label' => $object->label))
-    ->execute();
+  if ($object) {
+    return db_insert('islandora_usage_stats_objects')
+      ->fields(array('pid' => $pid, 'label' => $object->label))
```

Comment:
... Though yeah, given that it's [clearing out entries of objects on purge](https://github.com/Islandora/islandora_usage_stats/blob/bd745652411aabd3f5e7d9561880659009bdd907/islandora_usage_stats.module#L339-L353), there's something of an expectation not to be tracking non-existent objects... Hmm...